.travis.yml

```
@@ -4,6 +4,10 @@ sudo: required
 dist: trusty
 language: generic
 
+cache:
+  directories:
+    - $HOME/.ccache
+
```

```suggestion
cache: ccache
```
similar to https://github.com/PilzDE/pilz_industrial_motion/pull/254

---

according to https://docs.travis-ci.com/user/caching/#ccache-cache

---

@ct2034 the shorter form doesn't seem to work - see https://github.com/PilzDE/pilz_industrial_motion/pull/254#pullrequestreview-373562783

---

that seems odd.